introspection-engine/connectors/sql-introspection-connector/src/re_introspection.rs

```diff
@@ -217,6 +217,10 @@ pub fn enrich(old_data_model: &Datamodel, introspection_result: &mut Introspecti
             let enm = new_data_model.find_enum_mut(&changed_enum_value.0.enm);
             let value = enm.find_value_mut(&changed_enum_value.0.value);
             value.name = changed_enum_value.1.clone();
+            //fixme: pull out warnings for fixed values.
+            // introspection_result.warnings.retain(|w| )
```

Comment:
What kind of warnings would need to be filtered out here?

Reply:
Ah, that comment is obsolete now. It was referring to commenting out warnings, but since this now runs after re-introspection there should be no more surplus warnings. I'll get rid of the comment